### PR TITLE
New Swedish focus tree

### PR DESCRIPTION
**Pull request by MikaelLindmark**
political_advisor: more historical traits
New Swedish focus tree (up to middle of ww1)

### CODE REVIEW

**Comment by Wolferos** [comment]
Thank you for the PR! Did you run some tests to see how the new NF is handle by the game?

**Comment by MikaelLindmark** [comment]
Yes, I played three different scenarios, up to the middle of ww1. I wanted the focus tree to be closer to history and also include how close Sweden was with Germany. The "historical AI" should pick the correct path and there are no (extra) errors in the log.  I will add more to the focus tree later, together with some events. And probably some bug fixes. :)

**Comment by Wolferos** [comment]
Looks good! We'll run some tests on our own too but it shouldn't be a problem, thank you!

File: mod/thegreatwar/localisation/ww1_nf_sweden_l_english.yml [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+ SWE_ww1_equipment_effort:0 "Equipment Effort"
+ SWE_ww1_equipment_effort_desc:0 ""
+ SWE_ww1_doctrine_effort:0 "Doctrine Effort"
+ SWE_ww1_doctrine_effort_desc:0 ""
+ SWE_ww1_aviation_effort:0 "Aviation Effort"
+ SWE_ww1_aviation_effort_desc:0 ""
+ SWE_ww1_submarine_effort:0 "Submarine Effort"
+ SWE_ww1_submarine_effort_desc:0 ""
+ SWE_ww1_trenchwarfare:0 "Trench Warfare Focus"
+ SWE_ww1_trenchwarfare_desc:0 "Since Gatling and Maxim introduced their unholy weapons of warfare, our nation has been keenly interested in strategies based around the defense of Trench lines, which were first seen as far back as the American Civil War. We must seek experts in this theory in order to advance our knowledge in tomorrow's Land Doctrines."
```
Comment (NicolasGrosjean) [inline mod/thegreatwar/localisation/ww1_nf_sweden_l_english.yml:103]: Trench warfare existed before American Civil War.
[Source](https://en.wikipedia.org/wiki/Trench_warfare#Precursors).

**Pull request merged**